### PR TITLE
Define list of delegated methods in Worldwide Organisations

### PR DESCRIPTION
**Pull request by brucebolt**
We currently retrieve a list of methods that could be delegated to contacts in the Worldwide Organisation model, add some hardcoded methods, then remove a number of these.

As contacts haven't changed for many years, this dynamic code seems unnecessary and is more difficult to understand than if the methods were just hardcoded.

Therefore replacing with a list of methods. The delegation remains tested in the `WorldwideOfficeTest`.

Also removes a comment about "quacking" 🦆, which contains the wrong model name and also caused confusion as to what it actually meant when I first saw it.

### CODE REVIEW

File: app/models/worldwide_office.rb [hunk 1]
```diff
@@ -19,11 +19,23 @@ class WorldwideOffice < ApplicationRecord
...
+    country_name
+    email
+    has_postal_address?
+    locality
+    postal_code
+    recipient
+    region
+    street_address
+    title
+  ]
```
Comment (yndajas) [inline app/models/worldwide_office.rb:38]: Looks like this removes `country_id` - is that intentional?

Comment (brucebolt) [reply app/models/worldwide_office.rb]: I've added this back and updated the test to include all the delegated methods.

File: test/unit/app/models/worldwide_office_test.rb [hunk 2]
```diff
@@ -33,23 +33,22 @@ class WorldwideOfficeTest < ActiveSupport::TestCase
...
-    # associations
-    assert_equal contact.country, office.country
+    assert_equal contact.contact_form_url, office.contact_form_url
     assert_equal contact.contact_numbers, office.contact_numbers
-    # methods
+    assert_equal contact.contact_type_id, office.contact_type_id
     assert_equal contact.country_code, office.country_code
+    assert_equal contact.country_id, office.country_id
     assert_equal contact.country_name, office.country_name
+    assert_equal contact.country, office.country
```
Comment (yndajas) [inline test/unit/app/models/worldwide_office_test.rb:43]: These are in a slightly different order to the model (`country` is first there)

**Review by yndajas — APPROVED** [review]

File: test/unit/app/models/worldwide_office_test.rb (hunk 2 above)
Comment (brucebolt) [reply test/unit/app/models/worldwide_office_test.rb:43]: Looks like VSCode put them in that order as `_` comes before `,`.  I'll swap them round so it's consistent.

**Pull request merged**